app/views/devise/confirmations/new.html.erb

```diff
@@ -1,16 +1,18 @@
-<h2>Resend confirmation instructions</h2>
+<div class="margin-top"></div>
```

Comment:
Add a unique div to add a margin-top isn't a good practices, please added it in the correcto component